zerver/views/auth.py

```diff
@@ -35,6 +35,61 @@
 import time
 import ujson
 
+def find_my_team(request):
```

Comment:
I would put the forms code in zerver/forms.py maybe.  But you might want to ask on the "backend" channel of chat.